README.md

```diff
@@ -106,6 +113,8 @@ This command will retrieve the given package manager from the specified archive
 
 - `COREPACK_ENABLE_NETWORK` can be set to `0` to prevent Corepack from accessing the network (in which case you'll be responsible for hydrating the package manager versions that will be required for the projects you'll run, using `corepack hydrate`).
 
+- `COREPACK_NO_LOOKUP` can be set in order to instruct Corepack not to lookup on the remote registry for the latest version of the selected package manager.
```

Comment:
I'm trying to understand the difference between `COREPACK_ENABLE_NETWORK=0` and `COREPACK_NO_LOOKUP=1`? In theory I would expect no network to behave the same since corepack ships with Known Good Versions but I must be overlooking a use case here.

Reply:
When `COREPACK_NO_LOOKUP` is set, Corepack will use the version defined in `config.json` instead of asking the npm registry for the most up-to-date version. `COREPACK_ENABLE_NETWORK` can disable all access to the network, both for finding what's the most up-to-date version number and to download the tarball.

Reply:
Is `config.json` available to configure by the user of corepack or is it just internals?

Reply:
It's only internals, bundled in the corepack executable.

Reply:
How about COREPACK_IGNORE_LATEST?

Reply:
I went with `COREPACK_DEFAULT_TO_LATEST` which is true by default.

Reply:
Not sure about using default as a verb here. 

How about COREPACK_ENABLE_FETCH_LATEST=0

Reply:
I'd like the naming to make clear that it's only relevant if there's no local clue on what version to use. `ENABLE_FETCH_LATEST` could mean that the latest is always downloaded, or at least that's how I read it.

> Not sure about using “default” as a verb here.

I'm not a native english speaker, so genuine question: is it like incorrect to use "default" as a verb?

Reply:
It’s not incorrect, just that “default” can be a verb or a noun. It’s probably fine 👍